manual-tests.md

```diff
@@ -76,6 +76,15 @@
 5. Run `go-hello`.
 
 
+# Test cross-compilation with Rust
+
+1. Go to integration_tests/snaps/rust-hello.
+2. Run `snapcraft snap --target-arch=armhf`.
+3. Copy the snap to a Raspberry Pi.
```

Comment:
We could also probably help users without a Raspberry Pi test with qemu instructions.